ansible-roles/eks_cluster_init/tasks/main.yml

```diff
@@ -26,43 +26,52 @@
     - eks_cluster_init_eks_worker_iam_role_arn is defined
     - eks_cluster_init_k8s_admin_role_arn is defined
 
-- name: Create default encrypted StorageClass
-  kubernetes.core.k8s:
+- name: Get all StorageClasses
+  kubernetes.core.k8s_info:
+    kind: StorageClass
     api_version: storage.k8s.io/v1
+  register: storage_classes
+  environment: "{{ kubectl_env }}"
+
+- name: Remove default annotation from all StorageClasses
```

Comment:
It seems like this will always cause changes on every run.  Is it not possible to define the desired end state in one shot declaratively?  I assume this is to handle multiple possible existing start cases?  Could you add a new field to the new storage classes to make it possible to filter them from the old versions?